src/music.h

```diff
@@ -150,6 +154,8 @@ typedef struct
     /* Unload the library */
     void (*Unload)(void);
 
+    /* Start a specific track */
+    int (*StartTrack)(void *music, int track);
```

Comment:
Can you move `StartTrack` up after `GetNumTracks`?

Reply:
I think this is ready to merge after you make that change.

Reply:
Done!